### DISCUSSION TITLE
OpenDTU zeigt falsches GridProfile

### DISCUSSION THREAD

**Discussion by Taurus1958** [tbnobody/OpenDTU#2276] [Q&A]
Ich habe per Hoymiles DTU Pro das GridProfile "2.0.0 (AT_TOR_Erzeuger_cosphi=1)" auf meinen HM800 aufgespielt. In der S-Miles Cloud wird dieses richtig angezeigt, in der OpenDTU jedoch nicht! Gleiches Problem besteht auch bei meinem HM1500. Ich habe das GridProfile MEHRMALS neu aufgespielt jedoch ohne Wirkung. Was kann ich tun?
![GridProfileUnknown](https://github.com/user-attachments/assets/4301f55b-5706-4dc0-a7bb-4e41a949d1a8)

**Comment by stefan123t** [comment]
@Taurus1958 danke für die Frage. Wir haben evtl noch nicht die richtige Zuordnung der Grid Profile bei Änderungen durch die Hoymiles DTU.
Bitte den kompletten “Hash” des Grid Profiles (ganz unten) für alle Deine Inverter / Grid Profile im Issue #900 dokumentieren. Danke!

Das Setzen des Grid Profiles ist ausführlich auch in #900 diskutiert. Vielleicht hilft uns ja Dein Versuch mit der Hoymiles DTU den Unterschied zu finden und es auch bald in OpenDTU zu implementieren ?

**Comment by Taurus1958** [comment]
Hallo stefan123t,
habe die Werte kopiert und hier reingestellt, hoffe du hast dieses gemeint.
Ich bin mit solchen Sachen fast ein bissl überfordert, englisch ist auch nicht unbedingt meins ... da kommt dann mein Alter auch ein wenig zum Vorschein :-)
Der Name des Grid Profiles steht ja in meiner Anfrage.
![HM800](https://github.com/user-attachments/assets/3cc5d020-c095-49d5-8910-9da7ed1c75c7)
![HM1500](https://github.com/user-attachments/assets/b34d3104-72eb-4303-8557-42d40401efba)
